README.md

```diff
@@ -62,6 +62,8 @@ If you run the composer.phar with ```-v``` or ```-vv``` or ```-vvv``` it will sh
 
 If you want to extend an entity from your main application, you can use the ```entity-bundle-dir``` setting in the extra section of your composer.json.
 
+If you do not want to generate interfaces, you can set the ```generate-interfaces``` setting to ```false``` in the extra setting of your composer.json.
```

Comment:
This should guide people towards why they would, or would not, want to generate interfaces. That having them causes annoying backwards incompatibility.

It should denote whether we will enable the option by default in the future, so whether it's wise to already update.